quickwit/quickwit-indexing/src/actors/indexing_service.rs

```diff
@@ -508,7 +517,8 @@ impl IndexingService {
         for pipeline_id_to_remove in running_pipeline_ids.difference(&updated_pipeline_ids) {
             match self.detach_pipeline(pipeline_id_to_remove).await {
                 Ok(pipeline_handle) => {
-                    pipeline_handle.quit().await;
+                    // Killing the pipeline ensure that all pipeline actors will stop.
+                    pipeline_handle.kill().await;
```

Comment:
Use `kill`.

Reply:
Do we know it's safe to kill the pipeline? I worry about ongoing merges.

Reply:
Actually not sure what to test. The indexing pipeline refers to the merge planner mailbox... so killing the pipeline will not affect the merge planner.